test/integ/main_test.go

```diff
@@ -338,8 +339,7 @@ func TestRotator(t *testing.T) {
 	})
 }
 
-//nolint:tparallel
-// Subtests need to run in sequence. To parallelize this, but we'd need separate keys from the above (more cruft).
+//nolint:tparallel // Subtests need to run in sequence. To parallelize this, but we'd need separate keys from the above (more cruft).
```

Comment:
nit: 2 comments combined

Reply:
This is the lint format: https://golangci-lint.run/usage/false-positives/

Reply:
//nolint:gocyclo // This legacy function is complex but the team too busy to simplify it
func someLegacyFunction() *string {
  // ...
}